tfe/data_source_organization.go

```diff
@@ -0,0 +1,168 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEOrganization() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEOrganizationRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"external_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"collaborator_auth_policy": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"cost_estimation_enabled": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"enterprise_plan": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
🤔  I don't actually know what this is and don't see it in the API.

Reply:
I took this from the [go-tfe Organization struct](https://github.com/hashicorp/go-tfe/blob/master/organization.go#L83), but I should've checked the API as well. You're right, it isn't in the API.

Will remove this. Also, here is a PR in go-tfe to remove the unused attribute and add the ones that aren't there.

https://github.com/hashicorp/go-tfe/pull/225